### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by lincoln1883**
For this milestone, I completed the following task: 
- Create a app.rb file that will serve as your console app entry-point. It should have methods that do the following.
  - [x] List all books.
  - [x] List all people.
  - [x] Create a person (teacher or student, not a plain Person).
  - [x] Create a book.
  - [x] Create a rental.
  - [x] List all rentals for a given person id.
- In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:
  - [x] Present the user with a list of options to perform.
  - [x] Lets users choose an option.
  - [x] If needed, ask for parameters for the option.
  - [x] Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+  def list_all_rentals
+    puts 'ID of person: '
+    gets.chomp.to_i
+    puts 'Rentals: '
+    @rentals.select do |rental|
+      rental.person.id
+      puts "Date: #{rental.date} Book: #{rental.book.title} by #{rental.book.author}" "\n"
+    end
+  end
```
Comment (abdulhamiid) [inline app.rb]: I noticed that the method `def_list_all_rentals` is not working as expected. 🛠

Kindly note that the method is expected to list all rentals for a given `person id` not all the rented books in the database `@rentals = []` 🤓

i.e. If I request the `list of rentals` for a `user`, the method should return the list of books rented by that particular user **only**.

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @lincoln1883 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Build interactive console apps.
- ✔️ Highly organized/clean code
- ✔️ Use GitFlow


## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (lincoln1883) [reply app.rb]: Done.

Comment (Abenezer-Tilahun) [reply app.rb]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @lincoln1883 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Pull request merged**